docs/pages/webhook/events/AUCTION_BID_CREATED.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'AUCTION_BID_CREATED'
+---
+
+# AUCTION_BID_CREATED
+
+This event occurs whenever a bid on an auction is created.
+
```

Comment:
Could be nice to show how to import this type from the lib to try to prevent devs to copy paste it definition. Maybe something like:

## Usage  
```ts
import { Events, parseAndVerifyRequest } from '@nft/webhook'

const { data, type } = await parseAndVerifyRequest<'AUCTION_BID_CREATED'>(req, process.env.LITEFLOW_WEBHOOK_SECRET)
// data is type Events['BID_CREATED']
```

Reply:
good idea, done